src/main/resources/assets/bloodmagic/lang/fr_FR.lang

```diff
@@ -1,239 +1,250 @@
+
+item.BloodMagic.scribe.water.name=Craie d’Inscription Elémentaire: Eau
+item.BloodMagic.scribe.fire.name=Craie d’Inscription Elémentaire: Feu
+item.BloodMagic.scribe.earth.name=Craie d’Inscription Elémentaire: Terre
+item.BloodMagic.scribe.air.name=Craie d’Inscription Elémentaire: Air
+item.BloodMagic.scribe.dusk.name=Craie d’Inscription Elémentaire: Crépuscule
+item.BloodMagic.scribe.dawn.name=Craie d’Inscription Elémentaire: Aube
+
+item.BloodMagic.focus.weak.name=Focus de Téléposition
+item.BloodMagic.focus.enhanced.name=Focus de Téléposition Amélioré
+item.BloodMagic.focus.reinforced.name=Focus de Téléposition Renforcé
+item.BloodMagic.focus.demonic.name=Focus de Téléposition Diabolique
+
+item.BloodMagic.slate.blank.name=Tablette Vierge
+item.BloodMagic.slate.reinforced.name=Tablette Renforcée
+item.BloodMagic.slate.imbued.name=Tablette Imprégnée
+item.BloodMagic.slate.demonic.name=Tablette Diabolique
+item.BloodMagic.slate.ethereal.name=Tablette Éthérée
+
+item.BloodMagic.orb.weak.name=Orbe Sanguinaire Affaiblie
+item.BloodMagic.orb.apprentice.name=Orbe Sanguinaire de l’Apprenti
+item.BloodMagic.orb.magician.name=Orbe Sanguinaire du Magicien
+item.BloodMagic.orb.master.name=Orbe Sanguinaire du Maître
+item.BloodMagic.orb.archmage.name=Orbe Sanguinaire de l'Enchanteur
+item.BloodMagic.orb.transcendent.name=Orb Sanguinaire Transcendante
+
+item.BloodMagic.reagent.incendium.name=Incendium
+item.BloodMagic.reagent.magicales.name=Magicales
+item.BloodMagic.reagent.sanctus.name=Sanctus
+item.BloodMagic.reagent.aether.name=Aether
+item.BloodMagic.reagent.crepitous.name=Crepitous
+item.BloodMagic.reagent.crystallos.name=Crystallos
+item.BloodMagic.reagent.terrae.name=Terrae
+item.BloodMagic.reagent.aquasalus.name=Aquasalus
+item.BloodMagic.reagent.tennebrae.name=Tennebrae
+item.BloodMagic.reagent.offensa.name=Offensa
+item.BloodMagic.reagent.praesidium.name=Praesidium
+item.BloodMagic.reagent.orbisterrae.name=Orbis Terrae
+item.BloodMagic.reagent.virtus.name=Virtus
+item.BloodMagic.reagent.reductus.name=Reductus
+item.BloodMagic.reagent.potentia.name=Potentia
+
+item.BloodMagic.bloodShard.weak.name=Tesson Sanguin Faible
+item.BloodMagic.bloodShard.demon.name=Tesson Sanguin Diabolique
+
+item.BloodMagic.baseComponent.reagentWater.name=Réactif d'Eau
+item.BloodMagic.baseComponent.reagentLava.name=Réactif de Lave
+item.BloodMagic.baseComponent.reagentAir.name=Réactif d'Air
+item.BloodMagic.baseComponent.reagentFastMiner.name=Réactif de Miner
+item.BloodMagic.baseComponent.reagentVoid.name=Réactif du Vide
+item.BloodMagic.baseComponent.reagentGrowth.name=Réactif de Forestation
+item.BloodMagic.baseComponent.reagentAffinity.name=Réactif d'Affinitée Élémentaire
+item.BloodMagic.baseComponent.reagentSight.name=Réactif de Vue
+item.BloodMagic.baseComponent.reagentBinding.name=Réacitf de Sacrifice
+item.BloodMagic.baseComponent.reagentSuppression.name=Réactif de Délétion
+item.BloodMagic.baseComponent.frameParts.name=Parties des cadres
```

Comment:
If there are parts for a single frame, it would be "Parties de cadre" or "Morceaux de cadre"